frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -250,10 +257,24 @@ export const DetachedTestResultsList: any = ({
 
   const testResults = data?.testResults || [];
 
-  const rows = testResultRows(testResults, setPrintModalId, setMarkErrorId);
+  const rows = testResultRows(
+    testResults,
+    setPrintModalId,
+    setMarkErrorId,
+    setDetailsModalId
+  );
 
   return (
     <main className="prime-home">
+      {detailsModalId && (
+        <TestResultDetailsModal
+          testResultId={detailsModalId}
+          closeModal={() => {
+            setDetailsModalId(undefined);
+            refetch();
```

Comment:
Just curious why we're refetching here?

Reply:
This is copypasta from the print modal. I can (and probably should) remove it. I wonder if the print modal (erroneously) copied it from something that actually edited data and therefore would require a refetch.